_posts/2020-02-26-#17428.md

```diff
@@ -44,9 +50,92 @@ _Preliminary notes below. Additional notes will be published on Monday, February
+  is easier to map.
+
+- Other reasons anchoring all connections would be bad includes
+1. A victim of an eclipse attack has no way of escaping
+2. Strong persistence can contribute to the network self-partitioning. For example, if the longer
+  distance connections are less reliable, nodes are incentivized to connect
+  locally and this could lead to subgraphs per continent.
+3. An attacker could eclipse nodes with a small number of nodes but 100% uptime
+  and a large capacity for inbound connections. Every time a node adds a
+  connection, if its an adversary that malicious node would be "locked in"
+  forever with anchoring logic.
+
+- The suggestion of using blocks-relay-only peers mitigates the concerns around
+  reduced outbound peer rotation negatively impacting transaction privacy,
+  while maintaing the benefit of mitigating from an eclipse attack.
+
+### PR 17428
+- The 2 outbound block-relay-only connections are written to the `anchors.dat`
+  file. When a node restarts, it first tries establishing connections with the
+  anchors before attempting to connect to other peers.
+
+- Previously, a node that would quickly reattempt the same outgoing connections
+  was detected as a spy or mass connector. This new behavior means that honest
+  nodes will attempt the same outgoing connections more than once, which was
+  unlikely before.
+
+- The current state of this implementation introduces a new risk- if one of the
+  anchor peers is malicious and is able to exploit a DoS vulnerability, they
+  can repeatedly crash the victim's node.
+
+  The proposed fix is to forget the anchor peers on an unclean exit.
+
+  In reaction to these findings, the PR was [closed](https://github.com/bitcoin/bitcoin/pull/17428#issuecomment-583737058)
+  but then re-opened based on a comment saying there are still advantages of
+  preserving connections when restarting a node. The implementation of the
+  proposed fix is yet to come.
+
+
 ## Questions
 
-_Questions will be published on Monday, February 24._
+1. Did you review the PRs? [Concept ACK, approach ACK, tested ACK, or
+NACK?](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review)
+(Don't forget to put your PR review on GitHub.)
+
+2. What is an eclipse attack?
+  a. How does an adversary eclipse a node?
+  b. What attacks can an adversary execute on the victim?
+  c. How does this PR mitigate the attack? What are some limits?
+
+3. How does the anchors.dat file work?
+  a. When are peers persisted and removed?
+  b. When does the node ignore the contents of anchors.dat?
+  c. What are some implications of the timing of adding peers?
+
+4. What are the conditions for adding an anchor?
+  a. Why are the anchors blocks-only peers?
+  b. What would be the issue of using normal (transaction & block relay) peers
+  as anchors?
+  c. What are limitations of how this implementation chooses anchors?
+  d. Why limit to only 2 anchors?
```

Comment:
Questions 2, 3 and 4 might be more readable as bullet points or numbered markdown points.